src/init.ts

```diff
@@ -12,8 +12,17 @@ interface InitConfig extends Config {
 }
 
 export default function init( config: InitConfig, mw: MediaWiki ): ComponentPublicInstance {
+	const languageCode = mw.config.get( 'wgUserLanguage' ) as string;
+	const api = new mw.Api( {
+		parameters: {
+			formatversion: 2,
+			uselang: languageCode,
+			errorformat: 'html',
```

Comment:
I guess this should be ok, especially when looking at the alternative values here. We can look into this in detail when we actually come to that story.